ip: 192.168.0.20
camera_id: 4
frame_id: ipcam_4
H:
- - -0.0
  - -0.035714285714285636
  - -78.57142857142857
- - -0.07576144084141578
  - -0.17857142857142852
  - -1.5126778614938945
- - -2.168404344971009e-19
  - 0.0035714285714285696
  - 1.0
